Resources/Prototypes/Entities/Structures/Storage/filing_cabinets.yml

```diff
@@ -109,8 +109,8 @@
       - id: Pen
         prob: 0.5
       - id: PaperOffice
-      - id: PaperOffice
-        prob: 0.5
+        amount: 1
+        amountMax: 2
       - id: PaperOffice
         prob: 0.4
```

Comment:
at this point get rid of this and just make `amountMax` 3 and re-adjust the folder probability

Reply:
should the `prob` sum be exactly 1.0?

Reply:
for the folders? yeah

Reply:
Can I just remove `prob` and expect that probability will be equally divided within the group?

Reply:
that would just make all folders always spawn since prob is by default 1

Reply:
Oh no, I just noticed that I wrote `amountMax ` instead `maxAmount`. I'll fix it asap

Reply:
![Screenshot_1](https://user-images.githubusercontent.com/100279397/215284425-6a8b7a39-f960-4f6e-89f7-49fd941c4110.png)
I have also found that removing probability works just fine with orGroup.